.github/workflows/main.yml

```diff
@@ -95,3 +95,11 @@ jobs:
         run: |
           cd ethtool
           cargo test
+
+      - name: test (mptcp-pm)
+        env:
+          # Needed root permission to modify MPTCP
+          CARGO_TARGET_X86_64_UNKNOWN_LINUX_GNU_RUNNER: "sudo -E"
+        run: |
+          cd mptcp-pm
+          cargo test
```

Comment:
nit: pre-existing but we could use `cargo test -p` option or [`working-directory` keyword of GHA](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstepsrun).

Reply:
Fixed with `cargo test -p`. Thanks.